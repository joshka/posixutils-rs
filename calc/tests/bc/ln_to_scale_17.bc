a = l(0.2)
b = l(0.6)
c = l(2)
d = l(14.9867)
e = l(130.230890)
f = l(1302.9012834)

scale = 17
a/1
b/1
c/1
d/1
e/1
f/1
quit
