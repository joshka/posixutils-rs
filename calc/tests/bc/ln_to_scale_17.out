-1.60943791243410037
-0.51082562376599068
0.69314718055994530
2.70716314111414067
4.86930895201907172
7.17234881324178779
